jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/oqua/Question.kt

```diff
@@ -45,17 +47,20 @@ data class Question(
     override fun hashCode(): Int = Objects.hash(start, end, question, answer)
 
     companion object {
-        fun getQuestionsFromChunk(chunk: Chunk): List<Question> {
+        fun getQuestionsFromChunk(chunk: Chunk): Single<MutableList<Question>> {
```

Comment:
Most of the time we don't need to return a mutable list from a function. Also, if we don't really need mutable collection, then we don't use it.